modules/client/test/helpers/deployContracts.ts

```diff
@@ -166,6 +168,24 @@ export async function deploy(): Promise<Deployment> {
       .connect(deployOwnerWallet)
       .attach(addressListRepoAddr);
 
+    const multisigFactory = new aragonContracts
+      // @ts-ignore
+      // TODO update contracts-ethers
+      .MultisigSetup__factory();
+    const multisigPluginSetup = await multisigFactory
+      .connect(deployOwnerWallet)
+      .deploy();
+    const multisigRepoAddr = await deployPlugin(
+      pluginRepoFactory,
+      multisigPluginSetup.address,
+      "Multisig",
+      [1, 0, 0],
```

Comment:
What are these 3 values? Multisig only has `minApproval` count now

Reply:
thats the version